managed-databases/document-database/how-to/renew-tls-certificate.mdx

```diff
@@ -0,0 +1,49 @@
+---
+meta:
+  title: How to renew a TLS certificate for a Database Instance
+  description: This page explains how to renew a TLS certificate for a Database Instance
+content:
+  h1: How to renew a TLS certificate for a Database Instance
+  paragraph: This page explains how to renew a TLS certificate for a Database Instance
+tags: managed-database database postgresql ferretdb mongodb document database-instance
+dates:
+  validation: 2023-08-01
+  posted: 2023-08-01
+categories:
+  - managed-databases
+---
+
+The creation and management of TLS certificates is a feature included in the Managed Document Database offers.
+
+A certificate is generated automatically upon the creation of a Database Instance. You can renew your certificates at any time.
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have a [Document Database Instance](/managed-databases/document-database/quickstart)
+</Message>
+
+1. Click **Document Databases** under **Managed Databases** on the side menu. A list of your Database Instances displays.
+2. Click the database name or <Icon name="more" /> > **More info** to access the Database Instance information page.
+3. Click **Renew** under **TLS Certificate** in the **Database Instance Information** section. A pop-up appears.
+  <Lightbox src="scaleway-renew-button.webp" alt="" />
+
+  A pop-up appears.
+
+  <Lightbox src="scaleway-rdb-renew-tls-popup.webp" alt="" />
+
+4. Click **Renew TLS certificate** once you have read the warnings displayed and wish to continue.
+
+    <Message type="important">
+      Renewing a certificate means you:
+        - Will not be able to connect to your Database Instance with the previous certificate when using the [verify-ca and verify-full](/managed-databases/document-database/api-cli/verify-ca-postgresql).
```

Comment:
remove "when using the verify-ca and verify-full" or completely the first bullet point.
Indeed, mongodb-compatible clients enforce the certificate ca verification by default, which means client won't be able to connect at all with the previous certificate (whereas by default postgresql clients do not perform this check unless "verify-ca" or "verify-full" options are use to connect.